packages/openapi-generator/src/parser/operation.ts

```diff
@@ -72,13 +72,33 @@ function isPlaceholder(pathPart: string): boolean {
   return /^\{.+\}$/.test(pathPart);
 }
 
+function validatePlaceholder(pathPart: string): boolean {
```

Comment:
Could you add some small statement what this method checks with an example. The regex is so hard to read.

Reply:
The argument is called pathPart but it is called with placeholder. Rename?